pandas/_libs/tslibs/timezones.pyx

```diff
@@ -385,24 +385,26 @@ def tz_standardize(tz: tzinfo) -> tzinfo:
     -------
     tzinfo
 
-    Examples:
+    Examples
     --------
+    >>> from datetime import datetime
+    >>> from pytz import timezone as tz
+    >>> tz = tz('US/Pacific').normalize(datetime(2014,1,1, tzinfo=pytz.utc)).tzinfo
     >>> tz
     <DstTzInfo 'US/Pacific' PST-1 day, 16:00:00 STD>
-
     >>> tz_standardize(tz)
     <DstTzInfo 'US/Pacific' LMT-1 day, 16:07:00 STD>
 
+    >>> from pytz import timezone
+    >>> tz = timezone('US/Pacific')
     >>> tz
     <DstTzInfo 'US/Pacific' LMT-1 day, 16:07:00 STD>
-
     >>> tz_standardize(tz)
     <DstTzInfo 'US/Pacific' LMT-1 day, 16:07:00 STD>
 
-    >>> tz
+    >>> tz        # doctest: +SKIP
     dateutil.tz.tz.tzutc
-
-    >>> tz_standardize(tz)
+    >>> tz_standardize(tz)    # doctest: +SKIP
```

Comment:
why does this need skipping?

Reply:
I am not able to replicate this example

```python
In [33]: from dateutil.tz.tz import tzutc as tz

In [34]: tz
Out[34]: dateutil.tz.tz.tzutc

In [35]: from pandas._libs.tslibs.timezones import tz_standardize

In [36]: tz_standardize(tz)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-36-94d2ff882b92> in <module>
----> 1 tz_standardize(tz)

TypeError: Argument 'tz' has incorrect type (expected datetime.tzinfo, got _TzSingleton)

Reply:
@jreback sorry to unresolve this, but can we remove this example if it's not possible to run it?